binary_transparency/firmware/.gitignore

```diff
@@ -0,0 +1,7 @@
+ft_personality
+ft_monitor
+flash_tool
```

Comment:
Not easily - golang doesn't care about your staging directory sensibilities, I thought about doing 'go run' everywhere rather than 'go build', but this integration test relies on the trillian one which does `go build` for trillian binaries.

Possibly fixable, but it's a bigger rabbit hole than I really want to go down at the mo'.